### DISCUSSION TITLE
[geo][location] (Geo Point Field) how ?

### DISCUSSION THREAD

**Discussion by StefanSa** [vectordotdev/vector#12943] [Q&A] [answered]
Hi there,
how do i get created with GeoIP transforms `location` (Geo Point Field) extracted / generate.
`location* | [geo][location] | {"lat": 47.6062, "lon": -122.3321}"`
https://www.elastic.co/guide/en/elasticsearch/reference/current/geo-point.html
example:
```
    "GeoLocation": {
      "country_name": "Germany",
      "location": {
        "lon": 9.491,
        "lat": 51.2993
      }
    },
```

**Comment by spencergilbert** [comment, marked as answer]
Sorry @StefanSa, I'm not sure I understand the question. The [`geoip`](https://vector.dev/docs/reference/configuration/transforms/geoip/) transform should put all the available data under a key (defaults to `geoip`) so you should be able to access it under the path `.geoip.<data>`

**Reply by spencergilbert** [reply]
Example for Kibana's usage of a `location` object: https://github.com/vectordotdev/vector/discussions/12943#discussioncomment-2872982

**Comment by StefanSa** [comment]
@spencergilbert 
In general, you're right.
But the important field `location` type `geo_ip` is missing, elementary for Kibana / opensearch map visualization.
```
      "location": {
        "lon": 9.491,
        "lat": 51.2993
      }
```
I need to create this field "location" from the existing longitude and latitude.
How do I do this most effectively ?

**Reply by spencergilbert** [reply]
```coffee
.location.lon = .geoip.longitude
.location.lat = .geoip.latitude
```

**Reply by StefanSa** [reply]
> 
thanks for the tip.
I could have come up with it myself.
